doc/locale/ja/LC_MESSAGES/server.po

```diff
@@ -449,6 +449,13 @@ msgstr ""
 "`--protocol` オプションとその引数により、サーバのプロトコルを指定することがで"
 "きます。\"memcached\"はmemcachedバイナリプロトコルを示しています。"
 
+msgid ""
+"You don't need to create table.  When groonga receive request, it create "
+"table automatically.  The table name is `Memcache` ."
```

Comment:
This table doesn't exist before accepting request?
If so, we should use "future tense" like this: This table will be `Memcache`.